app/js/components/Seller/MarginSelectorForm.jsx

```diff
@@ -28,7 +28,7 @@ class MarginSelectorForm extends Component {
                      className="form-control prepend"
                      value={margin}
                      onChange={(e) => this.props.marginChange(e.target.value)}
-                     validations={[required, isNumber]} />
+                     validations={[required, isNumber, lowerThan.bind(null, 100)]} />
```

Comment:
It could be higher than 100, just that no one with any right mind would do it

Reply:
Are you sure, that could avoid someone to make mistakes?

As you said, no one with any right mind would do it so why allow it?
Imagine I type 150 instead of 15

Reply:
Yeah ok I'll let you have it. We probably want to be greater or equal than 0 too though.

Reply:
Yep, that makes sense, will add that